src/StudioCore/Assets/Paramdex/ER/Defs/MapGridCreateHeightDetailLimitInfo.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PARAMDEF XmlVersion="2">
+  <ParamType>MAP_GRID_CREATE_HEIGHT_LIMIT_DETAIL_INFO_PARAM_ST</ParamType>
+  <DataVersion>2</DataVersion>
```

Comment:
Technically this would be dataversion 1, since it's the first iteration.
But fromsoft themselves have basically deprecated dataversion so I don't think anyone cares lol